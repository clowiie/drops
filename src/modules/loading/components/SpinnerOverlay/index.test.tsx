import React from 'react'

import { render } from '@testing-library/react'

import * as SpinnerOverlayType from '.'

describe('SpinnerOverlay', () => {
  const { default: SpinnerOverlay } = require('.') as typeof SpinnerOverlayType

  describe('snapshots', () => {
    it('should match snapshots', () => {
      const { container } = render(<SpinnerOverlay />)

      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
