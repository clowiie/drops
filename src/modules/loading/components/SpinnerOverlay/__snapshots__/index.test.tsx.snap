// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerOverlay snapshots should match snapshots 1`] = `
<div
  class="fixed inset-0 flex justify-center items-center bg-black bg-opacity-40"
>
  <div
    class="static bg-white rounded-md outline-none p-2"
  >
    <div
      class="sc-bdnxRM ijGDfq rounded-full w-14 h-14 animate-spin"
    />
  </div>
</div>
`;
